lib/SMS.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+
+class SMS
+
+    TIME_FORMAT = '%H:%M'
+
+  #attr_reader :client, :config
+
+  def initialize(config, client: nil)
+    @client = client || Twilio::REST::Client.new(config[:account_sid], config[:auth_token])
+    @config = config
+  end
+
+  def deliver
+    client.messages.create(message_args)
+  end
+
+private
+
+attr_reader :client, :config
+
+  def message_args
+    {
+       from: config[:from],
+       to: config[:to],
+       body: config[:body] % delivery_time
+    }
+  end
+
+  def delivery_time
```

Comment:
I really like that you split each aspect of the SMS class out. I can't spot if you did env variables - did you get that far?